GitUI/BranchTreePanel/RepoObjectsTree.Nodes.Branches.cs

```diff
@@ -281,8 +281,11 @@ private Nodes FillBranchTree(IEnumerable<string> branches, CancellationToken tok
                 #endregion
 
                 var nodes = new Nodes(this);
+
+                token.ThrowIfCancellationRequested();
                 var aheadBehindData = _aheadBehindDataProvider?.GetData();
 
+                token.ThrowIfCancellationRequested();
                 var currentBranch = Module.GetSelectedBranch();
```

Comment:
❔ Why not pass `token` to `Module.GetSelectedBranch()`?

Reply:
Should be done in  #7447, too.